cmd/snap-update-ns/change.go

```diff
@@ -392,6 +392,7 @@ func (c *Change) lowLevelPerform(as *Assumptions) error {
 		}
 		return err
 	case Keep:
+		as.AddChange(c)
```

Comment:
I really know little about this code so my comment maybe silly - but when I read this I was a bit surprised. It reads as "Keep:" but the first thing it does is adding a change which seems strange. Maybe a comment? The commit comment explains it but a bit. Maybe something like: `// keep track of reused entries for the trespassing algorithm` ?